.cirrus.yml

```diff
@@ -15,6 +15,10 @@ test_linux_task:
   env:
     PATH: "${CIRRUS_WORKING_DIR}/otp/bin:${PATH}"
 
+
+  # don't cancel the task execution if it's master or a release branch
+  auto_cancellation: $CIRRUS_BRANCH != 'master' && $CIRRUS_BRANCH !=~ 'v\d+\.\d+.*'
```

Comment:
Is auto cancellation reported as a build failure? If not, I don't mind the auto-cancelling. It may make it a bit harder to detect when a build failed but at the same time, it saves resources.

Reply:
By default, autocancellation is disabled for the main branch (master), what this PR does is trear the vX.X branches as master.

Reply:
It is treated as aborted (RED CROSS), let me check if it's reported back to github as failure or as not run.

Reply:
See the screenshot, they are marked with a red cross as not sucessful, inside the report they are marked with a grey "!" symbol, indicating they have been skipped.

![image](https://user-images.githubusercontent.com/9133420/69546178-ce292b80-0fc5-11ea-8eed-d67b7eb4bb62.png)